editorconfig.go

```diff
@@ -122,44 +117,19 @@ var (
 	regexpBraces = regexp.MustCompile("{.*}")
 )
 
-func filenameMatches(pattern, name string) bool {
-	// basic match
-	matched, _ := filepath.Match(pattern, name)
-	if matched {
-		return true
-	}
-	// foo/bar/main.go should match main.go
-	matched, _ = filepath.Match(pattern, filepath.Base(name))
-	if matched {
-		return true
-	}
-	// foo should match foo/main.go
-	matched, _ = filepath.Match(filepath.Join(pattern, "*"), name)
-	if matched {
-		return true
-	}
-	// *.{js,go} should match main.go
-	if str := regexpBraces.FindString(pattern); len(str) > 0 {
-		// remote initial "{" and final "}"
-		str = strings.TrimPrefix(str, "{")
-		str = strings.TrimSuffix(str, "}")
-
-		// testing for empty brackets: "{}"
-		if len(str) == 0 {
-			patt := regexpBraces.ReplaceAllString(pattern, "*")
-			matched, _ = filepath.Match(patt, filepath.Base(name))
-			return matched
-		}
+// normalize fixes some values to their lowercaes value
+func (d *Definition) normalize() {
+	d.Charset = strings.ToLower(d.Charset)
+	d.EndOfLine = strings.ToLower(d.EndOfLine)
+	d.IndentStyle = strings.ToLower(d.IndentStyle)
 
-		for _, patt := range strings.Split(str, ",") {
-			patt = regexpBraces.ReplaceAllString(pattern, patt)
-			matched, _ = filepath.Match(patt, filepath.Base(name))
-			if matched {
-				return true
-			}
-		}
+	// tab_width defaults to indent_size:
+	// https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#tab_width
+	num, err := strconv.Atoi(d.IndentSize)
+
```

Comment:
I'd delete this empty line since the condition below is related to the values assigned above.

```suggestion
```